docs/toolbar.md

```diff
@@ -0,0 +1,54 @@
+---
+id: toolbar
+title: Toolbar
+---
+
+![Toolbar component](assets/toolbar.png)
+
```

Comment:
Lets add here some short description what it is, to make sure everybody understand the difference between TabBar and Toolbar

Reply:
@souhe I've just added a simple description. Is that good enough?

Reply:
👍